src/api/ZoweExplorerApi.ts

```diff
@@ -43,6 +43,15 @@ export namespace ZoweExplorerApi {
          * @returns {Session} a Zowe CLI Session
          */
         getSession(profile?: IProfileLoaded): Session;
+
+        /**
+         * Create a session for the specific profile type.
+         *
+         * @param {IProfileLoaded} profile
+         *      will use the profile the API was retrieved with by default
+         * @returns {IZosmfInfoResponse} z/OSMF Check Status response
+         */
+        getZosmfStatus(profile: IProfileLoaded): Promise<zowe.IZosmfInfoResponse>;
```

Comment:
The API is intended to be independent of zosmf and abstract from it.  So the method be called something generic such as `validateProfile()`.

Reply:
Plus we probably need to define a new return type interface. So far all the other return types were zosmf independent such as IZosFilesResponse.  We would need something like IZosServerInfo.